static/src/stylesheets/module/content/_recipe-article.scss

```diff
@@ -0,0 +1,439 @@
+
+$grey:#dbdbdb;
+$grey-two:#f6f6f6;
+$grey-three:#767676;
+$purple:#B72166;
+
+$padding-wide: 100px;
+
+
+.recipe__article .recipe__article--headline h1 {
+    font-family: $f-serif-headline;
```

Comment:
We have font mixins here: https://github.com/guardian/frontend/blob/d4422b4537165424e70a898d150db4e806ba04d6/static/src/stylesheets/pasteup/typography/_src.scss. Please use those instead